app/views/admin/site_steps/_footer.html.erb

```diff
@@ -6,6 +8,14 @@
         <% unless local_assigns[:no_continue] %>
           <%= f.button Admin::SiteStepsController::CONTINUE, value: Admin::SiteStepsController::CONTINUE, type: 'submit', method: :put, class: 'c-button -outline -dark-text' %>
         <% end %>
+        <% if step == 'style' %>
+          <%= react_component('AdminPreviewButton', {
+            text: Admin::SiteStepsController::PREVIEW,
+            className: 'c-button -outline -dark-text',
+            host: Rails.env.development? ? 'http://localhost:3000' : ENV['API_URL'],
```

Comment:
This is an error. The `API_URL` variables points to the RW API, not to the server running the app. As commented above, the React component can directly use `window.location` to read the base URL.